services/wireguard/connection/connection.go

```diff
@@ -80,6 +81,12 @@ func (c *Connection) Start(options connection.ConnectOptions) (err error) {
 		return err
 	}
 
+	if err := c.waitHandshake(); err != nil {
+		c.stateChannel <- connection.NotConnected
+		c.connection.Done()
+		return errors.Wrap(err, "failed to wait peer handshake")
```

Comment:
```suggestion
		return errors.Wrap(err, "error while waiting for peer handshake")
```

Reply:
Is it a good idea to start error with a `error` word? Looks like overkill for me.

Reply:
in logs usually you do not see that it is an error.(depending on logger), so I dont see why not to state that it is an error.

Reply:
There is a `state` in the logger. And this error not for logging, it for the upper layer, it could be anything.

Reply:
not `state` sorry
`log level`: `error` `info` `warn` `debug` etc